docs/notebooks/Toroidal_current_constraint.ipynb

```diff
@@ -0,0 +1,837 @@
+{
```

Comment:
<p>Should regenerate the current plot after PR #440</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>